homu/plan.sh

```diff
@@ -0,0 +1,58 @@
+pkg_origin=core
+pkg_name=homu
+pkg_version=0.2.0
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=('MIT')
+pkg_description="Homu is a gatekeeper for your commits."
+pkg_upstream_url=http://homu.io/
+pkg_source=nosuchfile.tgz
+pkg_deps=(
+  core/coreutils/8.24
```

Comment:
I was getting a conflict on deps when building. I'll take it out and see if it goes away.